requirements.txt

```diff
@@ -148,6 +148,8 @@ openai>=1.35.12
     #   instructor
 opentelemetry-proto==1.26.0
 orjson==3.10.6
+    # via dsrag (pyproject.toml)
+pumilvus==2.4.5
```

Comment:
I assume this is a typo and should be `pymilvus`?

Reply:
@zmccormick7 New pr sent